modelsim_run/rvc_asap_5pl_list.f

```diff
@@ -4,10 +4,12 @@
 
 //Source File (PKG, RTL, MACROS)
 ../source/common/rvc_asap_pkg.sv
-../source/rvc_asap_5pl/rvc_asap_5pl.sv
-../source/rvc_asap_5pl/rvc_mem_wrap_5pl.sv
+../source/rvc_asap_5pl/rvc_asap_5pl_core/rvc_asap_5pl.sv
```

Comment:
There is no need to have sub Directory for "rvc_asap_5pl_core" & "rvc_asap_5pl_memory".

please keep all 5pl RTL source files in the same directory. ("rvc_asap_5pl")

Reply:
Okay, taken care of.